[stable/datadog] Fix trace agent and useConfigMap

ogmaresca opened the pull request:
#### Is this a new chart
No

#### What this PR does / why we need it:

Version 2.0.0 of the stable/datadog introduced new issues.

1. If `agents.useConfigMap` is set to to true, it causes a templating error because `.Values.image` no longer exists.

```
Error: template: datadog/templates/datadog-yaml-configmap.yaml:38:28: executing "datadog/templates/datadog-yaml-configmap.yaml" at <.Values.image.tag>: nil pointer evaluating interface {}.tag
```

2. If `agents.useConfigMap` is false and `agents.apm.enabled` is true, the Datadog trace agent container fails to startup with this error:

```
trace-agent not enabled. Set the environment variable DD_APM_ENABLED=true or add "apm_config.enabled: true" entry to your datadog.yaml. Exiting...
```

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

Commit:
[stable/datadog] Fix trace agent and useConfigMap

Signed-off-by: Gerard Godone-Maresca <ggodonemaresca+github@gmail.com>

lawliet89 committed:
[incubator/vault] Support additional selector for Vault service (#20995)

* [incubator/vault] Support additional selector for Vault service

Signed-off-by: Yong Wen Chua <lawliet89@users.noreply.github.com>

* Fix typo

Signed-off-by: Yong Wen Chua <lawliet89@users.noreply.github.com>

Commit:
[stable/osclass] Release 7.0.7 (#21117)

* [stable/osclass] Release 7.0.7 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/moodle] Release 7.2.4 (#21104)

* [stable/moodle] Release 7.2.4 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/jasperreports] Release 7.0.9 (#21119)

* [stable/jasperreports] Release 7.0.9 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/testlink] Release 7.1.4 (#21120)

* [stable/testlink] Release 7.1.4 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
Add Adnan and Nedim to OWNERS file (#21121)

Signed-off-by: Ciprian Hacman <ciprian.hacman@sematext.com>

alexsn committed:
[stable/prometheus-nats-exporter]: add prometheus-operator servicemonitor support (#21088)

Signed-off-by: Alex Snast <alexsn@fb.com>

Commit:
[stable/wordpress] Move WP to non-root (#21099)

* [stable/wordpress] Move WP to non-root

Signed-off-by: juan131 <juan@bitnami.com>

* [stable/wordpress] Update components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* README fixes and issue with validation

Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@vmware.com>

Co-authored-by: Bitnami Bot <containers-bot@bitnami.com>
Co-authored-by: Javier J. Salmerón-García <jsalmeron@vmware.com>

adberger committed:
[stable/prometheus] Use the new api for ingress (#21049)

Signed-off-by: Adrian Berger <adrian.berger@bedag.ch>

vsliouniaev committed:
[stable/prometheus-operator ] Bump operator version, update dependencies (#20572)

* Fix CRD annotations, bump operator version

Alter port on node-exporter because CI is down

Signed-off-by: vas <vasily.sliouniaev@jet.com>

* Update dependent charts to latest

Signed-off-by: vas <vasily.sliouniaev@jet.com>

* Update readme and cleanup-attempt template

Signed-off-by: vas <vasily.sliouniaev@jet.com>

rowanruseler committed:
[stable/pgadmin] README incorrect example (#20804)

Within the README file there was an incorrect example where env.username
was given rather then env.email.

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

salarmgh committed:
Update percona to 5.7.26 and fix manifest to use apps/v1 (#20164)

* Update percona to 5.7.26 and fix manifest to use apps/v1

Signed-off-by: Salar Moghaddam <salarmgh@gmail.com>

* Version bump

Signed-off-by: Salar Moghaddam <salarmgh@gmail.com>

unclenorton committed:
[stable/chartmuseum] Allow using initContainer to set volume permissions (#21019)

* [stable/chartmuseum] Use initContainer to set the permissions when securityContext is not available

Signed-off-by: Dmitry Kharchenko <unclenorton@gmail.com>

* [stable/chartmuseum] Use .Values.persistence.path for the PV mount path

Signed-off-by: Dmitry Kharchenko <unclenorton@gmail.com>

* [stable/chartmuseum] Correct the variable name in the readme

Signed-off-by: Dmitry Kharchenko <unclenorton@gmail.com>

* [stable/chartmuseum] Revert the changes to incubator chart

Signed-off-by: Dmitry Kharchenko <unclenorton@gmail.com>

hwoarang committed:
[stable/prometheus-adapter] Do not create custom metrics rbac (#20900)

Commit b7afaf9d8875f6aa1cfed4c0422cb28e51d823a3 ("do not create
apiservice when not needed") from PR #20883 disabled the custom API
Service when no custom rules are used. However, we also need to not
deploy the cluster role as it has no use.

Signed-off-by: Markos Chandras <markos@chandras.me>

tuananhnguyen-ct committed:
fix test race condition (#20967)

Signed-off-by: Tuan Anh Nguyen <tuananh.nguyen-ext@commercetools.de>

felippe-mendonca committed:
[stable/rabbitmq-ha] adds scrapeTimeout option on ServiceMonitor (#21093)

* [stable/rabbitmq-ha] adds scrapeTimeout option on ServiceMonitor

Signed-off-by: Felippe Mendonça <mendonca.felippe@gmail.com>

* [stable/rabbitmq-ha]: bumps chart version

Signed-off-by: Felippe Mendonça <mendonca.felippe@gmail.com>

bcg62 committed:
[stable/sysdig] fix setting env vars (#21106) (#21106)

Signed-off-by: Brendan Germain <brendan.germain@nasdaq.com>

Adri2000 committed:
[stable/prometheus] Added configmapReload.enabled parameter (#20578) (#20912)

* Added configmapReload.enabled parameter (#20578)

Signed-off-by: Adrien Cunin <adrien.cunin@springernature.com>

* Splitted configmapReload parameters

It's now possible to independently change the parameters of the two
configmapReload containers using configmapReload.prometheus.* and
configmapReload.alertmanager.* parameters.

Signed-off-by: Adrien Cunin <adrien.cunin@springernature.com>

* Bumped chart version

Signed-off-by: Adrien Cunin <adrien.cunin@springernature.com>

RachadAbiChahine committed:
[stable/pgadmin] Pr modifies pgadmin to accept enhanced_cookie_protection  env to fix ingress issue #18568 (#21045)

* False and True should be forced otherwise it will be interpreted as boolean

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

* updates enhanced_cookie_protection

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

* adds the documentation of enhanced_cookie_protection in pgadmin

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

* as discussed I upgrade to 1.2.2

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

* Updates README.md to adds default env variables

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

* the value of enhanced_cookie_protection should be double quoted

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

* update README.md : the value of enhanced_cookie_protection should be double quoted

Signed-off-by: Rachad Abi Chahine <rachad.abi-chahine@hotmail.fr>

Commit:
Added fullname as prefix for init-scripts configmap name. (#21110)

* Added fullname as prefix for init-scripts configmap name.

Signed-off-by: Marcos Monteiro <marcos.monteiro@greenbricklabs.com>

* Update Chart.yaml

Signed-off-by: Marcos Monteiro <marcos.monteiro@greenbricklabs.com>

chrpme committed:
Upgrade to latest release 1.3.0 (#21101)

Signed-off-by: Mike Unke <unke@b1-systems.de>

blakeroberts-wk committed:
update eventrouter to v0.3 (#21140)

Signed-off-by: blakeroberts-wk <blake.roberts@workiva.com>

ahmadalli committed:
[stable/minio] annotation fix (#21155)

* annotation fix
Signed-off-by: ahmadali shafiee <mail@ahmadalli.net>

* use with operator
Signed-off-by: ahmadali shafiee <mail@ahmadalli.net>

Commit:
Use ksm chart for ksm (#21148)

By leveraging the ksm chart we are deduplicating community effort.

Signed-off-by: Naseem <naseem@transit.app>

askhalil committed:
chore: use the new api for ingress (#20648)

Signed-off-by: Abdulrahman Khalil <a.skhalil93@gmail.com>

ahmetkaftan committed:
[stable/mongodb] updateStrategy added when useStatefulSet is true and serviceName added. (#21150)

* updateStrategy added when useStatefulSet is true and serviceName added.

Signed-off-by: Ahmet Kaftan <ahmet.kaftan@cloudnesil.com>

* typo fix.

Signed-off-by: Ahmet Kaftan <ahmet.kaftan@cloudnesil.com>

Commit:
[stable/wordpress] Fix issue with port number when using external dbs (#21172)

Signed-off-by: juan131 <juan@bitnami.com>

ogmaresca committed:
[mssql/linux] Allow setting tolerations, affinity, and securityContext (#20745)

Signed-off-by: Gerard Godone-Maresca <ggodonemaresca+github@gmail.com>

devOpsHelm committed:
Add runAsGroup in SecurityContext (#101) (#21173)

Signed-off-by: devOpsHelm <devops+1@hazelcast.com>

devOpsHelm committed:
Add runAsGroup in SecurityContext (#101) (#21174)

Signed-off-by: devOpsHelm <devops+1@hazelcast.com>

jjaniec committed:
[stable/sealed-secrets] Add podLabels support (#21083)

Signed-off-by: Joffrey Janiec <jjaniec@student.42.fr>

ryuheechul committed:
[stable/nginx-ingress] match minReplicas to the documentation (#21184)

Signed-off-by: Heechul Ryu <ryuhcii@gmail.com>

jrob committed:
[stable/influxdb] Additional auth job variables (#21075)

* Adding hookDeletePolicy variable.

Signed-off-by: Jeremy Roberts <jeremy@robertsisland.com>

* Adding backoffLimit variable.

Signed-off-by: Jeremy Roberts <jeremy@robertsisland.com>

* Bumping chart version.

Signed-off-by: Jeremy Roberts <jeremy@robertsisland.com>

devOpsHelm committed:
[stable/hazelcast-jet] Merge pull request #93 from gurbuzali/jet-4.0 (#21182)

* Merge pull request #93 from gurbuzali/jet-4.0

Signed-off-by: devOpsHelm <devops+1@hazelcast.com>

* Fixed linter error

Signed-off-by: Emin Demirci <emin@hazelcast.com>

Co-authored-by: Emin Demirci <emin@hazelcast.com>

Commit:
[stable/osclass] Release 7.0.8 (#21143)

* [stable/osclass] Release 7.0.8 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/testlink] Release 7.1.5 (#21145)

* [stable/testlink] Release 7.1.5 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/ghost] Release 9.1.11 (#21192)

* [stable/ghost] Release 9.1.11 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/moodle] Release 7.2.5 (#21152)

* [stable/moodle] Release 7.2.5 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodríguez Hernández <carrodher1179@gmail.com>

Commit:
[stable/sysdig] Upgrade to latest agent version (9.6.1) (#21213)

Signed-off-by: Néstor Salceda <nestor.salceda@sysdig.com>

Commit:
[newrelic-infra] Bump chart to latest version (#21187)

Signed-off-by: Lennert Claeys <lclaeys@newrelic.com>

javsalgar committed:
[stable/postgresql] Add sidecars support (#21188)

* [stable/postgresql] Add sidecars support

Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@vmware.com>

* Fix if

Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@vmware.com>

Commit:
[stable/jenkins] update pluging versions (#21205)

Signed-off-by: Torsten Walter <torsten.walter@syncier.com>

Yatharth0045 committed:
[stable/consul] updated templating for ingress host (#21207)

Signed-off-by: Yatharth Sharma <yatharth0045@gmail.com>

samcv committed:
[stable/nextcloud] add subPath option (#21210)

* [stable/nextcloud] add subPath option

[Why]
The user will now be able to specify nextcloud.persistence.subPath and
it will automatically set each of the nextcloud-data mounts properly

Signed-off-by: Samantha McVey <samantham@posteo.net>

* [stable/nextcloud] bump chart version to 1.9.2

Signed-off-by: Samantha McVey <samantham@posteo.net>

* [stable/nextcloud] document persistence.subPath option

Signed-off-by: Samantha McVey <samantham@posteo.net>

Commit:
Update README.md (#19876)

bump version

Signed-off-by: David Karlsen <david@davidkarlsen.com>

maltekrupa committed:
[stable/prometheus] Support tmpl files which are not YAML (#21208)

Signed-off-by: Malte Krupa <github-nqyroppg@nafn.de>

stevie- committed:
[stable/airflow] be able to change git sync target (#21217)

Helpful when you want to sync to a subdir

Signed-off-by: Steffen Tautenhahn <stevie-@users.noreply.github.com>

devOpsHelm committed:
[stable/hazelcast-jet] Merge pull request #103 from eminn/security-context (#21219)

* Merge pull request #103 from eminn/security-context

Signed-off-by: devOpsHelm <devops+1@hazelcast.com>

* Fixed linter error

Signed-off-by: Emin Demirci <emin@hazelcast.com>

Co-authored-by: Emin Demirci <emin@hazelcast.com>

dpkirchner committed:
[stable/traefik] Add distinct container SecurityContext tunable (#20743) (#20887)

* [stable/traefik] Add distinct container SecurityContext tunable (#20743)

Signed-off-by: David Kirchner <dpk@dpk.net>

* [stable/traefik] Update new securityContext K8S API links to something more up to date

Signed-off-by: David Kirchner <dpk@dpk.net>

Commit:
Make liveness and readiness probes configurable (#19637)

Signed-off-by: Maarten de Waard <maarten@greenhost.nl>

bradrydzewski committed:
[stable/drone] Add deprecation note in favor of official Charts (#21151)

* [stable/drone] Add deprecation note in favor of official Charts

Signed-off-by: Brad Rydzewski <brad.rydzewski@gmail.com>

* bump chart version number

Signed-off-by: Brad Rydzewski <brad.rydzewski@gmail.com>

* remove maintainers to pass linting

Signed-off-by: Brad Rydzewski <brad.rydzewski@gmail.com>

joceluss committed:
Allow configuring terminationGracePeriodSeconds (#21072)

Signed-off-by: Jocelyn Lussier <jocelyn.lussier2@montreal.ca>

leogr committed:
[stable/falco] allow gRPC configuration using values (#21225)

* [stable/falco] allow gRPC configuration using values

Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

* [stable/falco] Update CHANGELOG.md

Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

* [stable/falco] chart version bump

Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

sbuzonas committed:
Use selectorLabels helper in headless service (#21196)

Signed-off-by: Steve Buzonas <steve@fancyguy.com>

ekeih committed:
[stable/redis-ha] Fix label selector of haproxy servicemonitor (#20209)

* [stable/redis-ha] Fix label selector of haproxy servicemonitor

The old servicemonitor did not actually match the deployed service,
because the service uses redis-ha.fullname as a label
(https://github.com/helm/charts/blob/master/stable/redis-ha/templates/redis-haproxy-service.yaml#L9)
but the servicemonitor matches redis-ha.name.

Signed-off-by: Max Rosin <m.rosin@syseleven.de>

* Raise chart version after rebase

Signed-off-by: Max Rosin <git@hackrid.de>

Commit:
Add deprecation notice. (#21231)

Signed-off-by: Naseem <naseem@transit.app>

Commit:
Add deprecation notice. (#21234)

Signed-off-by: Naseem <naseem@transit.app>

davidkarlsen committed:
[stable/sealed-secrets] Upgrade to v0.10.0 (#21238)

Signed-off-by: David Karlsen <david@davidkarlsen.com>

askhalil committed:
chore(grafana): update grafana version (#21214)

Signed-off-by: Abdulrahman Khalil <a.skhalil93@gmail.com>

devOpsHelm committed:
Merge pull request #104 from eminn/fsGroup-helm3-fix (#21239)

Signed-off-by: devOpsHelm <devops+1@hazelcast.com>

rgarcia89 committed:
[stable/cerebro] Add ability to set tls skip verify via vaules file (#21178)

* Update Chart.yaml

Signed-off-by: Raul Garcia <garcia@denic.de>

* Add ability skip tls verification

Signed-off-by: Raul Garcia <garcia@denic.de>

* Update configmap.yaml

fixed mixed behavior

Signed-off-by: Raul Garcia <garcia@denic.de>

* Update stable/cerebro/Chart.yaml

Signed-off-by: David Karlsen <david@davidkarlsen.com>

Co-authored-by: David J. M. Karlsen <david@davidkarlsen.com>

jfrabaute committed:
[stable/kube-state-metrics] Add missing permission for volumeattachments collector (#21079)

When enabling the volumeattachments collector, the following error is
logged without this permission:

Failed to list *v1.VolumeAttachment: volumeattachments.storage.k8s.io is forbidden: User "XXXXX:kube-state-metrics" cannot list resource "volumeattachments" in API group "storage.k8s.io" at the cluster scope

Signed-off-by: Fabrice Rabaute <fabrice@arista.com>

devOpsHelm committed:
make configurable securityContext at statefulset spec level (#102) (#21218)

Signed-off-by: devOpsHelm <devops+1@hazelcast.com>

a1kemist committed:
[stable/elastic-stack] update requirements for 1.16 (#18620)

Signed-off-by: a1kemist <lsnelson33@gmail.com>

oba11 committed:
[stable/mongodb-replicaset] feature: add support for extra containers and volumes (#20875)

* add support for extra containers and volumes

Signed-off-by: Oluwaseun Obajobi <oba@obajobi.com>

* fix: updated the pull request with recommended changes

Signed-off-by: Oluwaseun Obajobi <oba@obajobi.com>

* update the chart version to `3.12.0`

Signed-off-by: Oluwaseun Obajobi <oba@obajobi.com>

k8s-ci-robot committed:
[sonatype-nexus] provide granular resource requests to nexus backup container (#21123)

Signed-off-by: Jeff Knurek <j.knurek@travelaudience.com>

k8s-ci-robot committed:
[stable/prometheus-operator] Add ThanosRuler resources to ClusterRole (#21190)

* Add ThanosRuler resources to ClusterRole

Signed-off-by: Steve Buzonas <steve@fancyguy.com>

* Bump chart version

Signed-off-by: Steve Buzonas <steve@fancyguy.com>

k8s-ci-robot committed:
fix hard-coded path and default to "/" (#21253)

* fix hard-coded path and default to "/"

Signed-off-by: Joe Steffee <jsteffee@flexera.com>

* bump version

Signed-off-by: Joe Steffee <jsteffee@flexera.com>

k8s-ci-robot committed:
[stable/kube-state-metrics] Enable all collectors by default (#21245)

Signed-off-by: Manuel Rüger <manuel@rueg.eu>

k8s-ci-robot committed:
Deprecate sonatype-nexus chart (#21255)

Signed-off-by: Rob Kernick <rjkernick@gmail.com>

k8s-ci-robot committed:
[stable/cluster-autoscaler] reuse existing service account (#21204)

* [stable/cluster-autoscaler] reuse existing service account

Signed-off-by: Nicolas Degory <nicolas.degory@gmail.com>

* refactor service account name in values.yaml

Signed-off-by: Nicolas Degory <nicolas.degory@gmail.com>

k8s-ci-robot committed:
[prometheus/stable] auto import of dashboards with helm template(helm 3) (#21263)

* [prometheus/stable] auto import of dashboards with helm template(helm 3)

Signed-off-by: gowrisankar <gowrisankar.m01@sap.com>

* Fix the chart version

Signed-off-by: gowrisankar <gowrisankar.m01@sap.com>

Co-authored-by: gowrisankarm <gowrisankar.m01@sap.com>

k8s-ci-robot committed:
[stable/kube-state-metrics] Only create PSP RBAC if rbac.create is true. (#19986)

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

k8s-ci-robot committed:
[stable/prometheus-redis-exporter] Fix secret key path in deployment (#21259)

* Update secret key path in deployment

Signed-off-by: Ken Thompson <ken.thompson23@gmail.com>

* Clarify use of SSL connections in redis address

Signed-off-by: Ken Thompson <ken.thompson23@gmail.com>

* Update chart version

Signed-off-by: Ken Thompson <ken.thompson23@gmail.com>

k8s-ci-robot committed:
[stable/elasticsearch-curator] load envs from secrets first (#20874)

* [stable/elasticsearch-curator] load envs from secrets first

allows using environment variable composition with env
variables that are loaded from secrets (e.g. for loading
a HTTP password and then compositing it into a full
basic auth env variable).

Signed-off-by: Marvin Beckers <marvin.beckers@cerence.com>

* stable/elasticsearch-curator: bump version

Signed-off-by: Marvin Beckers <marvin.beckers@cerence.com>

k8s-ci-robot committed:
[stable/memcached] fixes updateStrategy for Deployment (#18989)

* [stable/memcached] fixes updateStrategy for Deployment

Signed-off-by: Raffael Sahli <sahli@gyselroth.com>

* bump chart version

Signed-off-by: Raffael Sahli <sahli@gyselroth.com>

* bump chart version

Signed-off-by: Raffael Sahli <sahli@gyselroth.com>

k8s-ci-robot committed:
[stable/prometheus-operator] add enforcedNamespaceLabel option to the prom operator (#21132)

Signed-off-by: Will Salt <williamejsalt@gmail.com>

k8s-ci-robot committed:
[stable/datadog] Revert "Mount the directory containing the CRI socket instead of the socket itself" (#21268)

This reverts commit 24b4881ecb9ae7c4368359d59df4fcc968493356.

Fixes #21223

Signed-off-by: Lénaïc Huard <lenaic.huard@datadoghq.com>

k8s-ci-robot committed:
[stable/nginx-ingress] add instruction to make it work with nlb (#21260)

* add instruction to make it work with nlb

Signed-off-by: KongZ <goonohc@gmail.com>

* bump version

Signed-off-by: KongZ <goonohc@gmail.com>

* bump version

Signed-off-by: KongZ <goonohc@gmail.com>

* bump version

Signed-off-by: KongZ <goonohc@gmail.com>

k8s-ci-robot committed:
configurable upgrade strategy type (#21272)

The default is 'Recreate' in order to avoid issues with multi-attach
volmue errors

Signed-off-by: Jeff Billimek <jeffrey_k_billimek@homedepot.com>

k8s-ci-robot committed:
[stable/mongodb-replicaset] Namespace override (#21203)

Signed-off-by: Mohammad Forutan <mforutan@users.noreply.github.com>

k8s-ci-robot committed:
fix(values): move maxmindLicenseKey under controller (#21282)

Signed-off-by: azman0101 <azman0101@gmail.com>

k8s-ci-robot committed:
[stable/prometheus] Set rollingUpdate to null when strategy is Recreate (#21289)

This change improves handling of deployment strategy parameters:

1) Improve documentation on strategy parameters.

2) Document & use alertmanager.strategy.type, which was present in
   values.yaml but unused. Alertmanager was previous using server.strategy

3) Set strategy.rollingUpdate to null when strategy.type is 'Recreate'.
   This is necessary to prevent the following error on an upgrade
   switching the strategy type from 'RollingUpdate' to 'Recreate':

   ```
   UPGRADE FAILED: Deployment.apps "prometheus-server" is invalid:
   spec.strategy.rollingUpdate: Forbidden: may not be specified
   when strategy `type` is 'Recreate'
   ```

Signed-off-by: Alexandre Rebert <alex@forallsecure.com>

bdun1013 committed:
[stable/postgresql] Updated helper functions to use KubeVersion.Version instead of deprecated KubeVersion.GitVerson (#21236)

* [stable/postgresql] Setting kubernetes version to KubeVersion.Version and defaulting to deprecated KubeVersion.GitVersion

Signed-off-by: Brian Dunnigan <bdun1013dev@gmail.com>

* [stable/postgresql] Bumping chart version

Signed-off-by: Brian Dunnigan <bdun1013dev@gmail.com>

* [stable/postgresql] Added kubeVersion helper that works in Helm 2.x and 3.x

Signed-off-by: Brian Dunnigan <bdun1013dev@gmail.com>

* [stable/postgresql] Simplified kubeVersion helper function

Signed-off-by: Brian Dunnigan <bdun1013dev@gmail.com>

Commit:
[stable/suitecrm] Release 8.0.9 (#21278)

* [stable/suitecrm] Release 8.0.9 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Commit:
[stable/prestashop] Release 9.1.9 (#21270)

* [stable/prestashop] Release 9.1.9 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Commit:
[stable/phabricator] Release 9.0.11 (#21279)

* [stable/phabricator] Release 9.0.11 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Commit:
[stable/drupal] Release 6.2.9 (#21280)

* [stable/drupal] Release 6.2.9 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

titansmc committed:
stable/graylog Adding the possibility of specifying your own URI for the GeoLite DDBB location (#21267)

* Author: Josep Manel Andres Moscardo <titansmc@gmail.com>
Date:   Thu Mar 5 09:41:15 2020 +0100

 Changes to be committed:
	modified:   templates/configmap.yaml
	modified:   values.yaml

Signed-off-by: Josep Manel Andres Moscardo <titansmc@gmail.com>

* Author: Josep Manel Andres Moscardo <titansmc@gmail.com>
Date:   Thu Mar 5 09:46:15 2020 +0100

 Changes to be committed:
	modified:   templates/configmap.yaml
	modified:   values.yaml
Signed-off-by: Josep Manel Andres Moscardo <titansmc@gmail.com>

* Author: Josep Manel Andres Moscardo <titansmc@gmail.com>

Date:   Thu Mar 5 09:46:15 2020 +0100

 Changes to be committed:
	modified:   templates/configmap.yaml
	modified:   values.yaml
Signed-off-by: Josep Manel Andres Moscardo <titansmc@gmail.com>

	modified:   Chart.yaml
bumped Chart.yaml version

* Author: Josep Manel Andres Moscardo <titansmc@gmail.com>

Date:   Thu Mar 5 09:46:15 2020 +0100

Signed-off-by: Josep Manel Andres Moscardo <titansmc@gmail.com>
	modified:   README.md

* Author: Josep Manel Andres Moscardo <titansmc@gmail.com>

Signed-off-by: Josep Manel Andres Moscardo <titansmc@gmail.com>

marcostvz committed:
[stable/elasticsearch] Adds imagePullSecrets to test image (#21048)

Signed-off-by: Marcos Estevez <marcos.stvz@gmail.com>

yurrriq committed:
[stable/prometheus-operator] Fix roles for k8s >1.15 (#21261)

Signed-off-by: Eric Bailey <e.bailey@sportradar.com>

Uvedale committed:
[stable/fluentd] Enable use of statefulset without autoscaling (#19668)

Signed-off-by: Dale Tristram <dale.tristram@sprinthive.com>

tariq1890 committed:
[stable/external-dns]use the latest GH link of externaldns (#21291)

* [stable/external-dns]use the latest GH link of externaldns

Signed-off-by: Tariq Ibrahim <tariq181290@gmail.com>

* [stable/external-dns]use the latest GH link of externaldns

Signed-off-by: Tariq Ibrahim <tariq181290@gmail.com>

Commit:
Revert "[stable/postgresql] Updated helper functions to use KubeVersion.Version instead of deprecated KubeVersion.GitVerson (#21236)" (#21308)

This reverts commit 6b848abd21a2c8fd0acfba751282f404162a30e5.

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

clamoriniere commented on stable/datadog/templates/datadog-yaml-configmap.yaml:
```diff
@@ -35,7 +35,7 @@ data:
       max_cpu_percent: 0
 
-    {{- $version := (.Values.image.tag | toString | trimSuffix "-jmx") }}
+    {{- $version := (.Values.agents.image.tag | toString | trimSuffix "-jmx") }}
```
good catch

clamoriniere commented on stable/datadog/templates/container-trace-agent.yaml:
```diff
@@ -14,6 +14,8 @@
     - name: DD_LOG_LEVEL
       value: {{ .Values.agents.containers.traceAgent.logLevel | default .Values.datadog.logLevel | quote }}
+    - name: DD_APM_ENABLED
+      value: "true"
```
which agent version are you using? 
I think the default value is now `true` by default: https://github.com/DataDog/datadog-agent/blob/1c76b8381a195a0b0f629011a6225e936fe1d37a/pkg/config/config.go#L390

blairham replied:
I believe it to address this issue:

```bash
2020-03-07 00:16:32 UTC | TRACE | INFO | (pkg/util/log/log.go:482 in func1) | Loaded configuration: /etc/datadog-agent/datadog.yaml
2020-03-07 00:16:32 UTC | TRACE | INFO | (pkg/trace/agent/run.go:71 in Run) | trace-agent not enabled. Set the environment variable
DD_APM_ENABLED=true or add "apm_config.enabled: true" entry
to your datadog.yaml. Exiting...
```

blairham replied:
I can confirm that in 7.18.0-rc.6 it's on by default, however.

Commit:
[stable/rabbitmq-ha] Added 'topicPermission' key in definitions.json secret (#20656)

* Added the possibility of adding "topic_prmissions" key in definitions.json

Signed-off-by: Andrea Gilardoni <andrea.gilardoni96@gmail.com>

* Version Bump

Signed-off-by: Andrea Gilardoni <andrea.gilardoni96@gmail.com>

* Fixing version bump

Signed-off-by: Andrea Gilardoni <andrea.gilardoni96@gmail.com>

Commit:
Add deprecation notice. (#21232)

Signed-off-by: Naseem <naseem@transit.app>

Commit:
[stable/kapacitor] Chart deprecation (#21247)

* Add deprecation notice.

Signed-off-by: Naseem <naseem@transit.app>

* Deprecate chart.

Signed-off-by: Naseem <naseem@transit.app>

Commit:
[stable/drupal] Release 6.2.10 (#21297)

* [stable/drupal] Release 6.2.10 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Commit:
[stable/owncloud] Release 8.1.6 (#21299)

* [stable/owncloud] Release 8.1.6 updating components versions

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* Run "helm dependency update" using latest Helm 2

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Co-authored-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Commit:
Add deprecation notice. (#21233)

Signed-off-by: Naseem <naseem@transit.app>

Commit:
[stable/influxdb] Chart deprecation (#21246)

* Add deprecation notice.

Signed-off-by: Naseem <naseem@transit.app>

* Deprecate chart.

Signed-off-by: Naseem <naseem@transit.app>

rowanruseler committed:
[stable/pgadmin] deprecated (#21275)

* [stable/pgadmin] deprecated

Moved towards a new repo: https://github.com/rowanruseler/helm-charts

helm hub: https://hub.helm.sh/charts/runix/pgadmin4

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

* [stable/pgadmin] duplication of key description

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

Arno500 committed:
[stable/unifi] Adding captive portal service (#21241)

* [stable/unifi] Adding captive portal service
Signed-off-by: Arno Dubois <arno.du@orange.fr>

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

* [stable/unifi] Annnd bumping version
Signed-off-by: Arno Dubois <arno.du@orange.fr>

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

* Added an enabled switch

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

* [stable/unifi] Fixing feedbacks

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

* [stable/unifi] Adding captive portal ingress

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

* [stable/unifi] Better table formatting

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

* [stable/unifi] Fixed ingress

Signed-off-by: Arno DUBOIS <arnodubois@sweetpunk.com>

Co-authored-by: Arno DUBOIS <arnodubois@sweetpunk.com>

Commit:
Update to 2.0.3

Signed-off-by: Gerard Godone-Maresca <ggodonemaresca+github@gmail.com>

Commit:
Update CHANGELOG.md

Signed-off-by: Gerard Godone-Maresca <ggodonemaresca+github@gmail.com>
